CMSgov/mint-app
# [EASI-2606] Updating Truss package

# EASI-2606

## Changes and Description

- Update Truss package to 6.0.0
- Update dockerfile.frontend to use node 16.20.0
- update `.nvmrc` to use 16.20.0
- Updated `Alert` component to have headingLevel prop (default heading level has been deprecated, so it is now necessary to write it out)
- Update the way Icon is used in app (<Icon.{IconName} /> instead of <Icon{IconName} />)
- Updated SumamryBox to have sub-components  (`SummaryBoxHeading` and `SummaryBoxContent`)
- Replaced deprecated `Dropdown` component with `Select` component 

## How to test this change
1. `yarn install` and bring up environment
2. go through the app and ensure pages load and still match figma 
3. Check lint, test, and cypress tests pass



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
